contributing.md

```diff
@@ -49,9 +148,33 @@ Here are a few rules to keep in mind while writing a commit message
  Resolves: #123
  See also: #456, #789
 
+Do not forget to sign your commit! Use `git commit -s`
+
 ```
+
 This is taken from [chris beams git commit](https://chris.beams.io/posts/git-commit/).
 You may want to read this for a more detailed explanation (and links to other posts on
 how to write a good commit message).
 This content is licensed under [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/).
 
+###Continuous integration
+
+The [continuous integration](https://en.wikipedia.org/wiki/Continuous_integration) (CI) contains a set of automated tests that are triggered on a remote server. Some of them are immediately triggered when pushing new content to a PR (i.e. the tests hosted on TravisCI) while other need to be triggered by contributors of the project. This second set of tests can be manually triggered. To trigger them, write a command as a PR comment. The available commands are:
```

Comment:
Please wrap the lines at 88 characters.

Also the commands only work for members of the repository. By writing `contributors of the project` it sounds like everyone could do this since everyone can contribute.